index.html

```diff
@@ -319,6 +321,18 @@ <h2>Glossary</h2>
 
 
 
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="j">J</div>
+
+<p><dfn class="lint-ignore export">Jamo</dfn>. The basic unit used to form Hangul syllables, representing vowels and consonants in Korean.</p>
+
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="k">K</div>
+
+<p><dfn class="lint-ignore export">Kana</dfn>. <a href="https://unicode.org/glossary/#kana">Unicode definition</a>: <q cite="https://unicode.org/glossary/#kana">The name of a primarily syllabic script used by the Japanese writing system. It comes in two forms, <i>hiragana</a></i> and <i>katakana</a></i>. The former is used to write particles, grammatical affixes, and words that have no <i>kanji</i> form; the latter is used primarily to write foreign words.</q></p>
```

Comment:
There are 2 stray anchor end tags.  Note that our i18n checker will pull you up because there is no class name on the i tags – presumably that's a carry over from the Unicode text, but i don't think we need to retain their markup when we can do better.

Reply:
> There are 2 stray anchor end tags.

Removed.

> Note that our i18n checker will pull you up because there is no class name on the i tags – presumably that's a carry over from the Unicode text, but i don't think we need to retain their markup when we can do better.

Do you have any suggestions on how we should change it?

Reply:
Although I'm okay with defining hiragana and katakana in this document, I don't want to define Kanji here, because if we define Kanji, I think we should also define Hanzi, Hanja/Hancha, and maybe even Sawndip and Chữ Nôm. After consideration, I decided to point to Unicode's definition and not define these terms in this document for the time being.